tortoise/expressions.py

```diff
@@ -359,6 +359,8 @@ def _process_filter_kwarg(
             param = model._meta.get_filter(key)
 
         pk_db_field = model._meta.db_pk_column
+        if isinstance(model._meta.pk, OneToOneFieldInstance):
+            pk_db_field += "_id"
```

Comment:
Seeems a bit strange, that `db_pk_column` does not in fact store db column and we have to patch it in place

Did you try changing the way `db_pk_column` is defined? 
Or something else breaks if you change it there?

Reply:
Yes, this two lines are not readability. I will refactor later.

Reply:
I agree with Andrey, this looks like an issue which should be fixed in fields and models code, not in the expression code.